includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -4079,6 +4081,7 @@ class AMP_Allowed_Tags_Generated {
 				'tag_spec' => array(
 					'amp_layout' => array(
 						'supported_layouts' => array(
+							5,
```

Comment:
@schlessera Here's where the `container` layout is being added for `amp-list`. I'm not sure if it would change #4541, but FYI.

Reply:
Removed in b5b29780b after updating to updating to v2004142326360.